metadata-editor/app/model/UsageRightsProperty.scala

```diff
@@ -77,12 +77,9 @@ object UsageRightsProperty {
       publicationField(true),
       photographerField(externalPhotographersMap, "publication")
     )
-    case _:ContractPhotographer =>
-    List(
-      publicationField(false),
-      photographerField(
-        Random.shuffle(contractedPhotographers)
-              .take(2).map{case (photographer, _) => photographer}.mkString(", "))
```

Comment:
It was setting the example / placeholder text to:
`e.g. Steve Madocks, Suzy Gett`.